secretmanager/snippets/requirements.txt

```
@@ -1,2 +1,2 @@
-google-cloud-secret-manager==2.20.0
+google-cloud-secret-manager==2.21.1
 google-crc32c==1.5.0
```

Suggested workaround for https://github.com/googleapis/google-cloud-python/issues/13350 is to pin protobuf. Only applies to this sample in this PR. 


```suggestion
google-crc32c==1.5.0
protobuf==5.27.5 # https://github.com/googleapis/google-cloud-python/issues/13350#issuecomment-2552109593
```